BrainPortal/app/controllers/userfiles_controller.rb

```diff
@@ -258,9 +258,9 @@ def display
     begin
       if @viewer
         if params[:apply_div] == "false"
-          render :file   => @viewer.partial_path.to_s, :layout => false
+          render :file   => @viewer.partial_path.to_s, :layout => params[:apply_layout].present?
```

Comment:
This particular change is not related to Data Providers, but it can make the CivetQC viewers must nicer in cbrain-plugins-neuro (pull request pending).